src/pages/run-a-node.js

```diff
@@ -0,0 +1,1015 @@
+      body: ["page-run-a-node-sovereignty-1", "page-run-a-node-sovereignty-2"],
+    },
+  ]
+
+  const scrollToId = (id) => {
+    const element = document.getElementById(id)
+    if (!element) return
+    element.scrollIntoView({ behavior: "smooth", block: "start" })
+  }
+
+  return (
+    <GappedPage>
+      <PageMetadata
+        title={translateMessageId("page-run-a-node-title", intl)}
+        description={translateMessageId(
+          "page-run-a-node-meta-description",
+          intl
+        )}
+      />
+      <HeroContainer>
+        <Hero content={heroContent} isReverse />
+      </HeroContainer>
+
+      <Content>
+        <TwoColumnContent>
+          <Width60>
+            <h2>
+              <Translation id="page-run-a-node-what-title" />
+            </h2>
+            <h3>
+              <Translation id="page-run-a-node-what-1-subtitle" />
+            </h3>
+            <p>
+              <Translation id="page-run-a-node-what-1-text" />
+            </p>
+            <h3>
+              <Translation id="page-run-a-node-what-2-subtitle" />
+            </h3>
+            <p>
+              <Translation id="page-run-a-node-what-2-text" />
+            </p>
+            <h3>
+              <Translation id="page-run-a-node-what-3-subtitle" />
+            </h3>
+            <p>
+              <Translation id="page-run-a-node-what-3-text" />
+            </p>
+          </Width60>
+          <Width40>
+            <GatsbyImage image={getImage(data.hackathon)} />
+          </Width40>
+        </TwoColumnContent>
+      </Content>
+
+      <FlexContent>
+        <StyledExpandableInfo
+          image={getImage(data.impact)}
+          title={<Translation id="page-run-a-node-who-title" />}
+          contentPreview={<Translation id="page-run-a-node-who-preview" />}
+          background="runNodeGradient2"
+        >
+          <p>
+            <Translation id="page-run-a-node-who-copy-1" />
+          </p>
+          <p>
+            <Translation id="page-run-a-node-who-copy-2" />
+          </p>
+          <p>
+            <Translation id="page-run-a-node-who-copy-3" />
+          </p>
+          <h3>
+            <Translation id="page-run-a-node-who-copy-bold" />
+          </h3>
+        </StyledExpandableInfo>
+      </FlexContent>
+
+      <Content id="why-run-a-node">
+        <h2>
+          <Translation id="page-run-a-node-why-title" />
+        </h2>
+        <InfoGrid>
+          {whyRunANodeCards.map(({ image, title, preview, body }, idx) => (
+            <ExpandableCard
+              contentPreview={<Translation id={preview} />}
+              title={<Translation id={title} />}
+              svg={image}
+              key={idx}
+            >
+              {body.map((item, idx) => (
+                <p key={idx}>{<Translation id={item} />}</p>
+              ))}
+            </ExpandableCard>
+          ))}
+        </InfoGrid>
+      </Content>
+
+      <Divider />
+
+      <Content id="getting-started">
```

Comment:
Rather than:
- "Getting started - Part 1: Hardware"
- "Part 2: Software"

 I think I'd prefer a structure of:
- "Get started"
  - "Part 1: Hardware"
  - "Part 2: Software"

Reply:
@wackerow noting this still needs to be addressed.